include/image.mk

```diff
@@ -474,7 +474,7 @@ define Device/Build/initramfs
   $(KDIR)/$$(KERNEL_INITRAMFS_NAME):: image_prepare
   $(1)-images: $$(if $$(KERNEL_INITRAMFS),$(BIN_DIR)/$$(KERNEL_INITRAMFS_IMAGE))
   $(BIN_DIR)/$$(KERNEL_INITRAMFS_IMAGE): $(KDIR)/tmp/$$(KERNEL_INITRAMFS_IMAGE)
-	cp $$^ $$@
+	mv $$^ $$@
```

Comment:
As the ImageBuilder doesn't itself create initramfs images, doesn't that then mean that the image file placed there by the buildroot will be moved away on the first build and unavailable for future builds for the same profile?

Reply:
Did you test if this issue actually happens? Looking at Imagebuilder archives I don't see the KDIR_TMP folder being shipped and everything that's moved comes. from KDIR_TMP.

Reply:
I just tested this and it works just fine.